client/app/vps/cloud-database/order/vps-cloud-database-order.controller.js

```diff
@@ -0,0 +1,190 @@
+class VpsCloudDatabaseOrderCtrl {
+  constructor(
+    $q,
+    $timeout,
+    $translate,
+    $window,
+    atInternet,
+    CloudMessage,
+    OvhApiHostingPrivateDatabase,
+    OvhApiOrderPrivateDatabase,
+    OvhApiMe,
+  ) {
+    this.$q = $q;
+    this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.$window = $window;
+    this.atInternet = atInternet;
+    this.CloudMessage = CloudMessage;
+    this.ApiPrivateDb = OvhApiHostingPrivateDatabase.v6();
+    this.ApiOrderDb = OvhApiOrderPrivateDatabase.v6();
+    this.ApiMe = OvhApiMe.v6();
+  }
+
+  $onInit() {
+    this.loading = {
+      durations: false,
+      redirection: false,
+      purchaseOrder: false,
+    };
+
+    this.versions = [];
+    this.ramAmounts = [];
+    this.datacenters = [];
+    this.durations = null;
+
+    this.currentOrder = {
+      version: null,
+      ram: null,
+      datacenter: null,
+      duration: null,
+      contractsAccepted: false,
+    };
+
+    this.purchaseOrder = null;
+    this.redirectionTimeout = null;
+
+    this.loadSelectValues();
+    this.ApiMe.get().$promise.then((user) => {
+      this.user = user;
+    });
+  }
+
+  loadSelectValues() {
+    return this.ApiPrivateDb.availableOrderCapacities({ offer: 'public' }).$promise
+      .then((capacity) => {
+        this.versions = _(capacity.version).sortBy().map(version => ({
+          value: version,
+          name: this.$translate.instant(`common_database_version_${version.replace('.', '')}`),
+        })).value();
+        this.ramAmounts = _(capacity.ram).map(ram => ({
+          value: ram,
+          name: `${ram} ${this.$translate.instant('unit_size_MB')}`,
+        })).value();
+        this.datacenters = _(capacity.datacenter).map(datacenter => ({
+          value: datacenter,
+          name: this.$translate.instant(`common_datacenter_${datacenter}`),
+        })).value();
+      })
+      .catch(error => this.CloudMessage.error([
+        this.$translate.instant('vps_tab_cloud_database_order_fetch_capacities_failed'),
+        _(error).get('data.message', error),
+      ].join(' ')));
+  }
+
+  showOrRefreshDurations() {
+    this.cancelFurtherChoices();
+
+    if (!this.currentOrder.version
+            || !this.currentOrder.ram
```

Comment:
indentation ?

Reply:
Weird auto-indent from eslint I guess... I'll fix it.